src/platformdirs/unix.py

```diff
@@ -112,20 +112,21 @@ def user_log_dir(self) -> str:
     @property
     def site_data_path(self) -> Path:
         """:return: data path shared by users. Only return first item, even if ``multipath`` is set to ``True``"""
-        site_data_dir = self.site_data_dir
-        if self.multipath:
-            # If multipath is True, the first path is returned.
-            site_data_dir = site_data_dir.split(os.pathsep)[0]
-        return Path(site_data_dir)
+        return self._first_item_as_path_if_multipath(self.site_data_dir)
 
     @property
     def site_config_path(self) -> Path:
         """:return: config path shared by the users. Only return first item, even if ``multipath`` is set to ``True``"""
-        site_config_dir = self.site_config_dir
+        return self._first_item_as_path_if_multipath(self.site_config_dir)
+
+    def _first_item_as_path_if_multipath(self, directory: str) -> Path:
+        """
+        :return: first directory as path, even if ``multipath`` is set to ``True``
```

Comment:
no need to document a private method, so remove it (and if we did we'd need to document the argument too, not just the return type)